tests/jerry/es2015/typedarray-prototype-keys.js

```diff
@@ -0,0 +1,95 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+try {
+  var array = [Uint8Array, Uint16Array, Uint32Array, 
+               Float32Array, Float64Array, 
+               Int8Array, Int16Array, Int32Array];
+
+  array.forEach (function(e){
+    e.prototype.keys.call (undefined);
```

Comment:
This test is not correct. If the first (`Uint8Array`) `keys` call is performed and the expected output is a TypeError which is catched below then the second (and further) cases are not tested. The iteration of types should be outside of the try catch block and only the `..keys.call(..)` should be wrapped with try-catch. This is same for all test files.